From 510ec95457c227e7a710696e400e458fcffb36b4 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Mon, 20 Jan 2025 11:32:30 +0100
Subject: [PATCH] ARM: dts: stm32: remove OPP 400 on stm32mp15x

Remove OPP 400 MHz on stm32mp15x.
Make OPP 650 MHz compatible with all SOCs variants.

Change-Id: I13c7f533f067cc27d3efb7c678f37e65274779c4
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/429932
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Pascal PAILLET <p.paillet@st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm/boot/dts/st/stm32mp151.dtsi | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

--- a/arch/arm/boot/dts/st/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp151.dtsi
@@ -32,17 +32,10 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp-400000000 {
-			opp-hz = /bits/ 64 <400000000>;
-			opp-microvolt = <1200000>;
-			opp-supported-hw = <0x2>;
-			opp-suspend;
-		};
-
 		opp-650000000 {
 			opp-hz = /bits/ 64 <650000000>;
 			opp-microvolt = <1200000>;
-			opp-supported-hw = <0x1>;
+			opp-supported-hw = <0x3>;
 		};
 
 		opp-800000000 {
